src/main/java/teammates/common/util/Url.java

```diff
@@ -130,11 +81,11 @@ public String toString() {
     }
```

Comment:
I propose we have accessors `getBaseUrl()` `getRelativeUrl` `getAbsoluteUrl()`. The `toString()` should call `getAbsoluteUrl` because that is the most complete representation of the object.